### DISCUSSION TITLE
Feedback on Ollama+Ollama web ui issues

### DISCUSSION THREAD

**Discussion by LyingDoc** [open-webui/open-webui#2341] [General]
I installed a Docker image and used WebUI to associate it with the local server. After I successfully deployed it, for example, I retrieved llama3-7b from the Ollama library and asked questions on the Web-UI interface. If there were any problems, it would take a long time to respond and the generation process would be slow. What is the problem

**Comment by pkeffect** [comment]
I noticed this as well. My solution was to install Ollama locally and OpenWebUI in docker. This fixed that issue for me. I do not know what causes this issue other than just Ollama not liking the extra layer of virtualization in some setups.

**Reply by LyingDoc** [reply]
I deployed it using a 1 panel panel, but I don't know why. I searched for some GPUs that were mentioned as issues, but I found that the online evaluation speed was quite fast. However, the installation in Docker was slow and strange, and I don't know the specific problem
